news/2021-08-19-project-loved-august-2021.md

```diff
@@ -0,0 +1,283 @@
+
+Lasting 6 minutes, this gauntlet of a beatmap constantly tests your abilities as a player, whether it be with ridiculously fast jumpstreams, reading through the hybrid-oriented sections, or the ending bursts that peak at nearly 500 BPM. All things considered though, MyZterioN- has done a good job of not making the map feel overlayered, and they made sure to give players a number of breaks where it made the most sense.
+
+Give MyZterioN- your support by casting your vote!
+
+[![](/wiki/shared/news/2021-08-19-project-loved-august-2021/1085851.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Various Artists - 6K Endurance Patterns Practice Vol.1](https://osu.ppy.sh/beatmapsets/1085851#mania) by [Imperial Wolf](https://osu.ppy.sh/users/9080296), [\_IceRain](https://osu.ppy.sh/users/7859973), [\[Crz\]sunnyxxy](https://osu.ppy.sh/users/10333739), [Alipay](https://osu.ppy.sh/users/9339256), [Anson_98](https://osu.ppy.sh/users/6198587), [Arkman](https://osu.ppy.sh/users/8814590) and [tyrcs](https://osu.ppy.sh/users/13026904)\
+172 BPM, 5:48 | 5.18★ – 6.44★\
+The \[(marathon)\[DELETE UPON DOWNLOAD!\]\] difficulty is *not* being nominated for Loved.\
+*description written by [Penguinosity](https://osu.ppy.sh/users/10235296)*
+
+Imperial Wolf's collaborative 6K endurance pack is here! This pack contains beatmaps from \_IceRain, Aliplay, Arkman, \[Crz\]sunnyxxy, tyrcs, Anson_98, and of course, Imperial Wolf.
+
+As the title suggests, this pack is targeted towards more experienced 6K players since all of the maps here emphasize the "endurance" factor. Whether it's the relentless high BPM chordstreams of *Immortal saga* and *GuNGNiR*, or the maps that focus patterns on reading and coordination like *Eighth-slave* and *Dst\[ick\]orv*, what's here will give even some of the most experienced mania players a good challenge.
+
+The people who provided beatmaps to this set are well-known within the 6K mapping scene and the mapping community in general, and we're thankful that Imperial Wolf put this together as it's a pretty cohesive rice-biased pack with a good difficulty spread. Make sure to check out all of the mappers who've helped put this together, and make sure to send your vote!
+
+[![](/wiki/shared/news/2021-08-19-project-loved-august-2021/655302.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[YUC'e - Future Cider](https://osu.ppy.sh/beatmapsets/655302#mania) by [bsc](https://osu.ppy.sh/users/6928171), [Arzenvald](https://osu.ppy.sh/users/3027421), [Bpple](https://osu.ppy.sh/users/8210168) and [Jakads](https://osu.ppy.sh/users/259972)\
+180 BPM, 3:59 | 1.06★ – 7.37★\
+*description written by [\_underjoy](https://osu.ppy.sh/users/2235750)*
+
+*Future Cider* is a mapset that has been there for quite a long time, and it has even been showcased in the Finals Mappool of the [osu!mania 7K World Cup 2018](/wiki/Tournaments/MWC/2018_7K). Since then it has certainly become a crowd favourite, gaining significant traction.
```

Comment:
you either add "tournament" after or remove the "the" (or hypothetically remove "2018")

```suggestion
*Future Cider* is a mapset that has been there for quite a long time, and it has even been showcased in the Finals Mappool of [osu!mania 7K World Cup 2018](/wiki/Tournaments/MWC/2018_7K). Since then it has certainly become a crowd favourite, gaining significant traction.
```

Reply:
i'll remove "the". will fix in forum post commit.